org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/queues/processes/MonitorAtomProcessTest.java

```diff
@@ -115,7 +114,7 @@ public void testTermination() throws Exception {
 		pti.waitForBeanFinalStatus(5000l);
 		pti.checkLastBroadcastBeanStatuses(Status.TERMINATED, false);
 
-		Thread.sleep(100);
+		Thread.sleep(1000);
 //		TODO This is probably not a good way to test this as we can't guarantee what stage the terminate happens at
 //		assertEquals("Incorrect message after terminate", "Get value of 'thermocouple1' aborted (requested)", pti.getLastBroadcastBean().getMessage());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)